static/css/app.css

```diff
@@ -575,7 +536,7 @@ hr {
 .dbsearch input[type="search"],
 .rangeselector input[type="text"],
 .rangeselector input[type="search"] {
-  width: 98%;
+  width: 100%;
```

Comment:
I think the 98% before was a result of not using `box-sizing: border-box` before and needing to round down to padding. With border-box, it can be 100%.